apps/daimo-mobile/src/action/useSendAsync.ts

```diff
@@ -19,6 +23,9 @@ import { NamedError } from "../logic/log";
 import { getWrappedPasskeySigner } from "../logic/passkey";
 import { Account, getAccountManager, useAccount } from "../model/account";
 
+/** Action identifier, tuple of [opHash, txHash]. */
+export type ActionIdentifier = [Hex | undefined, Hex | undefined];
```

Comment:
> in doc comment, specify that exactly one of opHash, txHash should be set.

I think everything works just fine if both are set too (since they're both unique identifiers for an `OpEvent`) but added this explanation in doc comment and refactored `ActionIdentifier` -> `OpEventID` in common op.ts